app/code/dev/ProductComment/etc/module.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
+<module name="dev_ProductComment" setup_version="0.0.1">
```

Comment:
Please fix your module to start with capital "D" like "Dev_ProductComment" and fix XML file to be intended with 4 spaces.